### DISCUSSION TITLE
What are the plans for Ackee?

### DISCUSSION THREAD

**Discussion by mvdkleijn** [electerious/Ackee#380] [General]
I'm asking because I've seen little movement in the codebase since the latest release, which was well over a year ago.

I am currently using it for two sites and quite like it, so I'd love to know if there are any plans for Ackee development... or should I be looking for an alternative?

**Comment by electerious** [comment]
I'm not activate working on it right now, but I'm using it for my own sites, so I'm interested in keeping it working :)

**Reply by mvdkleijn** [reply]
Thanks for the response. I hope you will continue to work on it in the future. I quite like the UI and the fact that it is fairly easy and simple.

**Reply by merrywhether** [reply]
Would you consider adding a collaborator to help manage it? It would be nice to get some things like the Mongoose upgrade in to ensure continued compatibility with the latest version on Mongo Atlas.

**Reply by echoblag** [reply]
Bumping this thread, I would love to see this project continue and contribute to it.

Setting out some requirements and standards for contributors would be a welcome step, possibly even a roadmap that can everyone can work towards.

My background is in DevOps and would be happy to help out on that front.

**Reply by l0uisgrange** [reply]
Hi, this package is wonderful ! The UI is incredible and the interactivity 🤩.
I would love to contribute or that this package could just be maintained, because two years of inactivity seems a bit long 🥲.
Love your work for privacy and open source !
